### PR TITLE
STONEBLD-1758 don't pull the images

### PR DESCRIPTION
**Pull request by stuartwdouglas**
When creating manifest lists don't pull the images. The issue was that the docker image:tag@sha256:digest format requires buildah to pull the image. If we remove the tag section and just use the digest then the manifest list builds without needing to pull.

### CODE REVIEW

File: task/build-image-manifest/0.1/build-image-manifest.yaml [hunk 1]
```diff
@@ -46,7 +46,7 @@ spec:
     - name: TLSVERIFY
       value: $(params.TLSVERIFY)
   steps:
-  - image: quay.io/redhat-appstudio/buildah:v1.31.0@sha256:34f12c7b72ec2c28f1ded0c494b428df4791c909f1f174dd21b8ed6a57cf5ddb
+  - image: quay.io/redhat-appstudio/multi-platform-runner:01c7670e81d5120347cf0ad13372742489985e5f
```
Comment (chmeliik) [inline task/build-image-manifest/0.1/build-image-manifest.yaml:49]: What is the reason for this change?

File: task/build-image-manifest/0.1/kustomization.yaml [hunk 2]
```diff
@@ -1,5 +0,0 @@
-apiVersion: kustomize.config.k8s.io/v1beta1
-kind: Kustomization
-
-resources:
-- build-image-manifest.yaml
```
Comment (chmeliik) [inline task/build-image-manifest/0.1/kustomization.yaml:5]: These deleted files seem unrelated to this PR

**Review by chmeliik — COMMENTED** [review]
Nice! :+1:

**Review by tkdchen — COMMENTED** [review]
:+1:

File: task/build-image-manifest/0.1/build-image-manifest.yaml (hunk 1 above)
Comment (stuartwdouglas) [reply task/build-image-manifest/0.1/build-image-manifest.yaml:49]: oops, this was originally because I was going to use podman instead, which is part of that image, but it turned out that the issue was not with buildah, but rather with the use of tag + digest. I will revert.

File: task/build-image-manifest/0.1/kustomization.yaml (hunk 2 above)
Comment (stuartwdouglas) [reply task/build-image-manifest/0.1/kustomization.yaml:5]: We don't need these files. There has been a PR open to deleted them for weeks.

**Review by arewm — APPROVED** [review]
